pydss/flapper.py

```diff
@@ -64,9 +64,9 @@ class Bid :
 
 
     def __init__(self, vat, gem,msg) :
-        self.wards = dict()
-        self.message = msg
-        self.wards[msg.sender] = 1;
+
+        self.msg = msg
```

Comment:
we shouldn't be setting a `self.msg`, there's no need for this to be saved to the Flapper instance. all we need is `msg.sender`, and we only need it on line 96 (not in other functions), so you can pass in a `sender` parameter to the function, as per our convention